akka-actor/src/main/scala/akka/actor/dungeon/DeathWatch.scala

```diff
@@ -38,8 +39,10 @@ private[akka] trait DeathWatch { this: ActorCell =>
 
   override final def watchWith(subject: ActorRef, msg: Any): ActorRef = subject match {
     case a: InternalActorRef =>
+      if (subject.path.uid == 0)
+        println(s"# undefinedUid watchWith $self -> $subject ${subject.getClass.getName}") // FIXME
```

Comment:
I'll remove these printlns but want to verify if they are triggered by any of the tests first